### PR TITLE
Feature/complete i18n copy upgrade

### LINKED ISSUE #224: Complete i18n copy upgrade
The framework is in place, but the app requires all remaining copy to be implemented:

https://github.com/input-output-hk/cardano-explorer-app/tree/develop/source/features/i18n

### PR DESCRIPTION
**Pull request by DeeJayElly**
Fixes #224  - Feature/complete i18n copy upgrade

### CODE REVIEW

**Comment by rhyslbw** [comment]
Is this still WIP, and therefore not likely to need review yet?

**Comment by DeeJayElly** [comment]
> Is this still WIP, and therefore not likely to need review yet?



> Is this still WIP, and therefore not likely to need review yet?

Implemented @DominikGuzei @rhyslbw

File: source/features/i18n/translations/ja.json [hunk 1]
```diff
@@ -1,36 +1,133 @@
...
-    "pageTitle": "!!!Address | $t(productTitle)"
+    "addressLabel": "!!!Address",
+    "pageTitle": "!!!Address | $t(productTitle)",
+    "summaryAddressLabel": "!!!Address",
+    "summaryBalanceLabel": "!!!Final Balance",
+    "summaryTransactionsLabel": "!!!Transactions"
   },
   "block": {
-    "pageTitle": "!!!Block | $t(productTitle)"
+    "pageTitle": "!!!Block | $t(productTitle)",
```
Comment (rhyslbw) [inline source/features/i18n/translations/ja.json]: Missing `blocks`?

File: source/features/i18n/translations/de.json [hunk 2]
```diff
@@ -1,36 +1,133 @@
...
-    "pageTitle": "Addresse | $t(productTitle)"
+    "addressLabel": "!!!Address",
+    "pageTitle": "Addresse | $t(productTitle)",
+    "summaryAddressLabel": "!!!Address",
+    "summaryBalanceLabel": "!!!Final Balance",
+    "summaryTransactionsLabel": "!!!Transactions"
   },
   "block": {
-    "pageTitle": "Block | $t(productTitle)"
+    "pageTitle": "Block | $t(productTitle)",
```
Comment (rhyslbw) [inline source/features/i18n/translations/de.json]: Missing `blocks` ?

File: source/features/i18n/translations/ja.json (hunk 1 above)
Comment (DeeJayElly) [reply source/features/i18n/translations/ja.json]: Added

File: source/features/i18n/translations/de.json (hunk 2 above)
Comment (DeeJayElly) [reply source/features/i18n/translations/de.json]: Added

**Comment by DeeJayElly** [comment]
> The table header rows and footer links have been missed

Added. We have all the strings covered now.

**Comment by DeeJayElly** [comment]
> "an hour ago" for transactions is not translated:
> CleanShot 2020-03-06 at 22 00 57

Isn't this coming from dayjs() library? Can we even localize it?

**Comment by DominikGuzei** [comment]
@DeeJayElly i will look into it 👍

**Comment by DominikGuzei** [comment]
@rhyslbw @DeeJayElly i have added i18n support for day.js (for localized dates and ranges, it also supports Japanese, i double checked with Google Translate) 💪 

I also implemented most of the German translations (apart from the longer & legal text parts)

**Comment by DeeJayElly** [comment]
> The table header rows and footer links have been missed

Added

**Comment by DominikGuzei** [comment]
@DeeJayElly is adding Japenese translations (#225) to this PR

**Comment by DeeJayElly** [comment]
> @DeeJayElly is adding Japenese translations (#225) to this PR

Added

**Comment by DeeJayElly** [comment]
Can you guys approve this one and we can close it @rhyslbw @DominikGuzei

**Comment by DominikGuzei** [comment]
@rhyslbw in German you don't translate "Blockchain Explorer" because because we usually use the original terms when there is no good / common translation in our language available.

**Pull request merged**